lib/features/mailbox_dashboard/presentation/model/search/quick_search_filter.dart

```diff
@@ -89,6 +99,8 @@ enum QuickSearchFilter {
         return userProfile != null &&
           filter.from.contains(userProfile.email) &&
           filter.from.length == 1;
+      case QuickSearchFilter.sortBy:
+        return true;
```

Comment:
what is it?

Reply:
It is a flag for showing selected icon or default icon. About sort by filter, it always true when user apply search even if it is `Relevance`